tests/StatementTest.php

```diff
@@ -39,24 +39,21 @@ public function testInstantiation() {
 
     public function testFromJSONInvalidNull() {
         $this->setExpectedException(
-            'InvalidArgumentException',
-            'Invalid JSON: ' . JSON_ERROR_NONE
+            'TinCan\JsonParseErrorException'
         );
         $obj = Statement::fromJSON(null);
     }
 
     public function testFromJSONInvalidEmptyString() {
         $this->setExpectedException(
-            'InvalidArgumentException',
-            'Invalid JSON: ' . JSON_ERROR_NONE
+            'TinCan\JsonParseErrorException'
         );
         $obj = Statement::fromJSON('');
     }
 
     public function testFromJSONInvalidMalformed() {
         $this->setExpectedException(
-            'InvalidArgumentException',
-            'Invalid JSON: ' . JSON_ERROR_SYNTAX
+            'TinCan\JsonParseErrorException'
```

Comment:
Is this testing that the right error is returned in the right context? It doesn't look like it is since no parameter is passed?